packages/frontend/src/services/indexer/api.ts

```diff
@@ -79,7 +79,7 @@ export default {
                 // ---------------------
                 promises.push(
                     fetch(
-                        `${CONFIG.INDEXER_FASTNEAR_SERVICE_URL}/v0/public_key/${publicKey}`,
+                        `${CONFIG.INDEXER_FASTNEAR_SERVICE_URL}/v0/public_key/${publicKey}/all`,
```

Comment:
are they changing endpoint? what /all with & without

Reply:
They by default returns only 100 accounts and /all gives you all account ids

Reply:
Also, if I remember correctly - `/all` returns FUNCTION_CALL keys, the other endpoint doesn't